spec/facets/CatalogDatasetFacet.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://openlineage.io/spec/facets/1-0-0/CatalogDatasetFacet.json",
+  "$defs": {
+    "CatalogDatasetFacet": {
+      "allOf": [
+        {
+          "$ref": "https://openlineage.io/spec/2-0-2/OpenLineage.json#/$defs/DatasetFacet"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "technology": {
+              "description": "The storage framework for which the catalog is configured",
+              "type": "string",
+              "example": "iceberg, delta, hive"
+            },
+            "source": {
+              "description": "Source system where the catalog is configured.",
+              "type": "string",
+              "example": "spark, flink, hive"
+            },
+            "type": {
+              "description": "Type of the catalog.",
+              "type": "string",
+              "example": "jdbc, glue, polaris"
+            },
+            "uri": {
+              "description": "URI or connection string to the catalog, if applicable.",
```

Comment:
This description means that uri can be not applicable to some catalogs, but this field is required in JSON schema